pkg/controller/controller_utils_test.go

```diff
@@ -28,6 +28,8 @@ import (
 	"testing"
 	"time"
 
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
this is new and out of place

Reply:
(there shouldn't be the newline underneath so it gets automatically sorted and verified)

Reply:
I will goimports this file again